Lib/fontTools/designspaceLib/types.py

```diff
@@ -118,5 +133,9 @@ def getVFUserRegion(doc: DesignSpaceDocument, vf: VariableFontDescriptor) -> Reg
     # Any axis not mentioned explicitly has a single location = default value
     for axis in doc.axes:
         if axis.name not in vfUserRegion:
+            if not isinstance(axis.default, (int, float)):
```

Comment:
how can that be? default should not be optional for an axis

Reply:
The type checker says it's possible.

Reply:
well, then just assert it isn't the case